### PR TITLE
feat(conf) lua_socket_pool_size property in kong.conf

### PR DESCRIPTION
**Pull request by thefosk**
### Full changelog

* Added `lua_socket_pool_size` property in configuration file.

### CODE REVIEW

File: kong.conf.default [hunk 1]
```diff
@@ -322,3 +322,7 @@
 
 #lua_package_cpath =             # Sets the Lua C module search path
                                  # (LUA_CPATH).
+
+#lua_socket_pool_size = 30       # Specifies the size limit for every cosocket
```
Comment (thibaultcha) [inline kong.conf.default]: This line has a trailing whitespace, it's too bad

File: kong.conf.default [hunk 2]
```diff
@@ -322,3 +322,7 @@
 
 #lua_package_cpath =             # Sets the Lua C module search path
                                  # (LUA_CPATH).
+
+#lua_socket_pool_size = 30       # Specifies the size limit for every cosocket 
+                                 # connection pool associated with every remote
```
Comment (thibaultcha) [inline kong.conf.default]: This one too

**Comment by thefosk** [comment]
CHANGELOG updated in a separate PR.

**Comment by thefosk** [comment]
@thibaultcha updated.

**Pull request merged**